sdk/src/types/block/context_input/reward.rs

```diff
@@ -0,0 +1,82 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use core::str::FromStr;
+
+use derive_more::From;
+
+use crate::types::block::Error;
+
+/// A Reward Input is an input that indicates which transaction Input is claiming Mana rewards.
+#[derive(Clone, Copy, Eq, PartialEq, Hash, Ord, PartialOrd, From, packable::Packable)]
+pub struct RewardInput(u16);
```

Comment:
For consistency with our other enum variants, this should be called `RewardContextInput`

Reply:
Done